app.js

```diff
@@ -7,14 +7,13 @@ const path = require('path');
 const engine = require('ejs-locals');
 
 const app = express();
-const config = require('./config');
+const config = require('./config/config.example');
```

Comment:
never include a example file

Reply:
Yeah even i was thinking why it does have an example. Okay, i will change the file name.